docs/ref/class-based-views/base.txt

```diff
@@ -87,6 +87,12 @@ MRO is an acronym for Method Resolution Order.
 
         Performs key view initialization prior to :meth:`dispatch`.
 
+        Sets up three key instance attributes:
+
+        * ``self.request`` - The current request object
+        * ``self.args`` - Positional arguments captured from the URL pattern
+        * ``self.kwargs`` - Keyword arguments captured from the URL pattern
```

Comment:
```suggestion
        Assigns the :class:`~django.http.HttpRequest` to the view's ``request``
        attribute, and any positional and/or keyword arguments
        :ref:`captured from the URL pattern <how-django-processes-a-request>`
        to the ``args`` and ``kwargs`` attributes, respectively.
```
I think we can reuse some existing description from `as_view(**initkwargs)` above